AnkiDroid/src/main/java/com/ichi2/utils/Permissions.kt

```diff
@@ -172,6 +183,9 @@ object Permissions {
     fun Context.arePermissionsDefinedInAnkiDroidManifest(vararg permissions: String) =
         this.arePermissionsDefinedInManifest(this.packageName, *permissions)
 
+    /**
+     * Whether it would be possible to manager external storage (potentially after requesting permission).
```

Comment:
Other than this, consider rewriting to explain why this permission is different 

* Disabled on Google Play
* Requires a separate OS-level permission screen as it's a really dangerous permission

```suggestion
     * Whether it would be possible to manage external storage (potentially after requesting permission).
```